i3_quickterm/main.py

```diff
@@ -240,14 +240,18 @@ def launch_inplace(conf, shell):
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("-i", "--in-place", dest="in_place", action="store_true")
+    parser.add_argument("-c", "--config", dest="config", type=str, help="read config from specified file")
     parser.add_argument("shell", metavar="SHELL", nargs="?")
     parser.add_argument(
         "--version", action="version", version="%(prog)s {}".format(__version__)
     )
     args = parser.parse_args()
 
     conf = copy.deepcopy(DEFAULT_CONF)
-    conf.update(read_conf(conf_path()))
+    if args.config:
+        conf.update(read_conf(args.config))
```

Comment:
The config also has to be added to `qt_cmd` in `toggle_quickterm()` so that the correct configuration will be used in the child process' `launch_inplace()`.